### ISSUE TITLE
PR: Federated Export with wrong IfcGuids

### ISSUE THREAD

**Issue by denizmaral** [Autodesk/revit-ifc#825]
### Problem Description

When I export a model with its link to same project or same IfcSite, IfcGuids are not correctly written in Ifc file. ExportIFCUtils give me actually correct guid but in the file it is something else... 

![grafik](https://github.com/user-attachments/assets/5efd6229-4a3f-4ffe-9ae2-68b0204947f1)

[WrongGuid.zip](https://github.com/user-attachments/files/17829466/WrongGuid.zip)

### Revit Version

2024.0.x

### IFC for Revit Addon Version

24.x.x

### Windows Version

11 22H2

**Comment by denizmaral** [comment]
You are totally right. In that case I would love to see an overridden value, when I select a linked Element. 
Fyi: @andydandy74

**Comment by denizmaral** [comment]
I think it should be like that: If I have only one instance of the link => take the Guid that is shown in UI 
If there are more than one instances => override Guids in UI and take those. 
It's not that common, that we have many instances of a link.

**Comment by AngelVelezSosa** [comment]
There are two issues with the suggestions above.
1. An overridden value with you select a linked Element: at the moment, there is no way to assign a value to *one* copy of a linked Element, so it would be overridden for all of them, without some very complicated UI.
2. Use the GUID if there is only one link: that would work well in 90% of the cases, perhaps, but then:
a. Add link: get the "right" GUIDs.
b. Add a 2nd link: get "new" GUIDs.
c. Remove the first link: what happens?

What do people think?

**Comment by andydandy74** [comment]
Please close this issue. The problem is discussed in more detail in #880

**Comment by AngelVelezSosa** [comment]
Please see #880 .
